modules/standard/JSON.chpl

```diff
@@ -63,6 +63,37 @@ module JSON {
     }
 
     /* Serialize a value into a :record:`~IO.fileWriter` in JSON format */
+    /*
+      Serialize ``val`` with ``writer``.
+
+      Numeric values are serialized as though they were written with the format
+      as ``%i`` for integers and ``%er`` for ``real`` numbers. Please refer to
+      :ref:`the section on Formatted IO<about-io-formatted-io>` for more
+      information.
+
+      Booleans are serialized as the literal strings ``true`` or ``false``.
+
+      ``string`` values are serialized by wrapping the string in quotes, and
+      escaping quotes inside the string. ``bytes`` values are also wrapped
+      in quotes.
+
+      Enums are serialized using the name of the corresponding value. For example
+      with an enum like ``enum colors {red, green blue}``, the value ``red``
+      would be serialized as a string: ``"red"``.
+
+      The ``nil`` value and nilable class variables storing ``nil`` will be
+      serialized as the text ``null``.
+
+      Classes and records will have their ``serialize`` method invoked, passing
+      in ``writer`` and this Serializer as arguments. Please see the
+      :ref:`serializers technote<ioSerializers>` for more.
+
+      Classes and records are expected to implement the ``writeSerializable``
```

Comment:
Should this be adjusted to read as on: https://github.com/chapel-lang/chapel/pull/23502/files#diff-e54b013aa9de867488474be4a98980482efcc0038d15722b3796c7276913a84bR3201 ?

Reply:
I think this shorter version works fine for the two options. With deserialization it felt worth a longer sentence to be clear.